backend/src/v2/driver/driver.go

```diff
@@ -480,6 +480,28 @@ func extendPodSpecPatch(
 		podSpec.NodeSelector = kubernetesExecutorConfig.GetNodeSelector().GetLabels()
 	}
 
+	if tolerations := kubernetesExecutorConfig.GetTolerations(); tolerations != nil {
+		var k8sTolerations []k8score.Toleration
+
+		glog.Infof("Tolerations passed: %+v", tolerations)
+
+		for _, toleration := range tolerations {
+			if toleration != nil {
+				k8sToleration := k8score.Toleration{
+					Key:               toleration.Key,
+					Operator:          k8score.TolerationOperator(toleration.Operator),
+					Value:             toleration.Value,
+					Effect:            k8score.TaintEffect(toleration.Effect),
```

Comment:
This is an enum, what's the behavior if we pass a "randomstring" ?

Reply:
The workflow controller fails when it tries to create the pod:

```
Warning  WorkflowNodeError      6s    workflow-controller  Error node my-pipeline-xxlsx.root.comp.executor: task 'my-pipeline-xxlsx.root.com p.executor' errored: Pod "my-pipeline-xxlsx-1295791212" is invalid: spec.tolerations[0].operator: Unsupported value: "foo": supported values: "Equal", "Exists"
```

I think this ties back to the broader discussion of whether we want to reimplement validation or let the K8s API handle it.